docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,100 @@ keywords:
 
 :::
 
-Temporal Nexus is a feature of the Temporal platform designed to connect durable executions across team, Namespace, region, and cloud boundaries.*
-It promotes a more modular architecture for sharing a subset of your team’s capabilities through well-defined service API contracts for other teams to use, that abstract underlying Temporal primitives, like Workflows, or execute arbitrary code.
+## Connect Temporal Applications
+
+Nexus allows you to reliably connect Temporal Applications.
+Designed with Durable Execution in mind, Nexus enables each team to have their own Namespace for improved modularity, security, debugging, and fault isolation.
+
+Nexus Services can be used across and within Namespaces.
+Nexus Operations can span any length of execution, be synchronous or asynchronous, and be implemented with Temporal primitives, like Workflows, or execute arbitrary code.
+
+Unlike other forms of inter-service communication, Nexus combines a familiar programming model with the resiliency of the Temporal Platform and its queue-based Worker architecture.
+The result is an integrated Temporal experience with improved security, observability, and reliability.
+Nexus supports cross-team, cross-domain, cross-namespace, and multi-region use cases.
+
+## Why use Temporal Nexus?
+
+### Integrated developer experience
+
+- **Temporal SDKs**:
+  - Use Nexus Services from a caller Workflow.
+  - Run Nexus Service handlers in a Worker, often the same Worker as underlying primitives.
+  - Implement long-running asynchronous Nexus Operations as Workflows.
+  - Handle low-latency synchronous Nexus Operations with Temporal primitives or arbitrary code.
+  - Execute Operations with at-least-once semantics by default, and exactly-once semantics using Workflow ID reuse policies.
+- **Nexus Endpoints with a queue-based Worker architecture**:
+  - Connect callers and handlers through Nexus Endpoints, a reverse proxy for Nexus Services.
+  - Manage Endpoints in the Temporal Nexus Registry.
+  - Use an Nexus Endpoint by name, which routes to an upstream target Namespace and Task Queue.
+  - Serve requests from a Nexus Worker by polling an Endpoint's target Task Queue, with automatic load balancing.
+
+### Key features
+
+- **Built-in Temporal Nexus Machinery**:
+  - Reliable state-machine-based invocation and completion callbacks for Nexus Operations.
+  - Atomic handoff from Workflow Event History to Nexus Operation state machines.
+  - Automatic retries and circuit breaking.
+- **Integrated observability**:
+  - Operation lifecycle and error info in Workflow Event History.
+  - Bi-directional linking for end-to-end debugging across Namespaces.
+  - Metrics, traces, and logs.
+- **Improved blast radius isolation**:
+  - Each team can have its own Namespace.
+  - Per-Namespace security & rate limits.
+  - Isolate misbehaving Workers.
```

Comment:
Want to mention "Avoid leaking implementation details and cart blanche namespace access by exposing deliberate APIs"?

Maybe this can be worded a bit better.

Reply:
I guess it's mentioned below so probably don't need to repeat it here. For your consideration.

Reply:
incorporated